native/cocos/renderer/pipeline/custom/ArchiveFwd.h

```diff
@@ -27,9 +27,8 @@
  * The following section is auto-generated.
  * ========================= !DO NOT CHANGE THE FOLLOWING SECTION MANUALLY! =========================
  */
-// clang-format off
 #pragma once
-#include "cocos/base/std/variant.h"
+// clang-format off
```

Comment:
Why off clang format?

Reply:
These files are generated. clang format is turned off to prevent formatting by accident.
clang-format is only turned off in this file. Any file including these files still has clang-format.

Reply:
ok